compose.prod.yml

```
@@ -104,3 +104,8 @@ services:
       start_period: 60s
     networks:
       - rabbitmq
+
+networks:
+  webserver:
+  rabbitmq:
+  database:
```

```suggestion
  database:

```

ca marche si on donne juste les noms?

```yaml
networks:
  - webserver
  - rabbitmq
  - database
```

---

Nn j'ai essayé, il s'attend à avoir un objet